### PR TITLE
CUI-7384 : [Coral-Actionbar] Actionbar improvements for accessibility.

### PR DESCRIPTION
**Pull request by indra2gurjar**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: coral-component-actionbar/src/scripts/ActionBar.js [hunk 1]
```diff
@@ -556,7 +556,14 @@ class ActionBar extends BaseComponent(HTMLElement) {
   
     const primary = this._elements.primary;
     const secondary = this._elements.secondary;
-  
+
+    if(!primary.getAttribute('role')) {
```
Comment (icaraps) [inline coral-component-actionbar/src/scripts/ActionBar.js:560]: Can you please add space after `if` statement ?

Comment (indra2gurjar) [reply coral-component-actionbar/src/scripts/ActionBar.js]: Done.

**Comment by indra2gurjar** [comment]
@majornista Added role="menu" and "menuitem" as suggested.

**Comment by indra2gurjar** [comment]
@icaraps Done suggested changes. Please review.

**Pull request merged**